### PR TITLE
Add a script to make the fits files

### PR DESCRIPTION
**Pull request by jeanconn**
This works from an updated h5 source file and the old fits files

### CODE REVIEW

File: fits.py [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+    hdr = hdu[0].header
+    # Update AGASC VERSION
+    hdr['EXTVER'] = 17
+
+    for row in hdu[1].data:
+        # Set RSV1 to be -9999.0 by default (for most stars this is a no-op, already -9999.0)
+        row['RSV1'] = -9999.0
+        # Find the index of the AGASC ID we have in the sorted table of newdata
+        idx = np.searchsorted(newdata['AGASC_ID'], row['AGASC_ID'])
+        if newdata['AGASC_ID'][idx] != row['AGASC_ID']:
```
Comment (jeanconn) [inline fits.py]: I think this is a bit belt-and-suspenders, but I can't see a way that the end result here can be wrong.   The np.searchsorted should always be right, but if for any reason there were an incorrect ID in the *new* h5 file, you could end up with a mismatch.  The 'if' statement check seems not unreasonable, though perhaps it should just error out if there's a mismatch out of np.searchsorted.

**Comment by jeanconn** [comment]
There's now a copy of the fits file tree in /data/aca/agasc1p7 made from this script.   I've forgotten a bit of the history, but it looks like for Matlab support we'd made a tree of links to the files right in /data/agasc1p6, so I've done the same with this agasc1p7.  Seemed to work in my initial load up in SAUSAGE.

**Comment by jeanconn** [comment]
Note also that I use skare3/Py3 to make the files.  I think this is fine operationally.

**Comment by jeanconn** [comment]
@taldcroft  This doesn't have the Tycho2 proper motion fixes we wanted yet.  Will try to get this in today but not sure if it will make it.

https://github.com/sot/agasc/issues/23#issuecomment-414675015

**Comment by jeanconn** [comment]
Also, let me know if we should just hold off on the Tycho2 proper motion fixes, as I assume that will be more trouble for @malgosias to test.

**Comment by taldcroft** [comment]
I think you can just punt on the PM fixes for 1.7.

**Comment by jeanconn** [comment]
OK, let's wait for AGASC > 1.7 for that.  And yes, I'd prefer it perfect but it does seem like we've dealt with the consequences of the missing PM data and it would be more trouble to verify updated proper motion values in testing on both sides given limited time.

**Comment by taldcroft** [comment]
I don't think I noticed this is an entirely new repo.  Could this be in the `agasc` repo?  In 2 years I won't remember this repo exists.

**Comment by jeanconn** [comment]
Sure.  I thought the agasc repo was more the agasc module code but if it is all agasc stuff that's fine.

**Pull request merged**